packages/features/bookings/lib/handleCancelBooking.ts

```diff
@@ -453,7 +457,10 @@ async function handler(input: CancelBookingInput) {
     metadata: bookingToDeleteEventTypeMetadata,
   });
 
-  const eventManager = new EventManager({ ...bookingToDelete.user, credentials });
+  const eventManager = new EventManager(
+    { ...bookingToDelete.user, credentials },
+    bookingToDeleteEventTypeMetadata?.apps
+  );
```

Comment:
Pass the app metadata to the event manager so it can read the Salesforce settings on booking cancel.